include/ebpf_nethooks.h

```diff
@@ -151,6 +151,31 @@ typedef struct bpf_sock_addr
     uint64_t interface_luid; ///< Interface LUID.
 } bpf_sock_addr_t;
 
+#define SOCK_ADDR_EXT_HELPER_FN_BASE 0xFFFF
+
+typedef enum
+{
+    BPF_FUNC_sock_addr_get_current_pid_tgid = SOCK_ADDR_EXT_HELPER_FN_BASE + 1,
+} ebpf_sock_addr_helper_id_t;
+
+/**
+ * @brief Get current pid and tgid (sock_addr specific only).
+ *
+ * @param[in] ctx pointer to bpf_sock_addr_t context.
+ *
+ *
+ * @retval A 64-bit integer containing the current tgid and pid,
```

Comment:
```suggestion
 * returns A 64-bit integer containing the current tgid and pid,
```
Expanation: "retval" creates a 2-column name/description table where the first word goes in the name column and the rest goes in description.  That doesn't apply here.